### DISCUSSION TITLE
Names for what we are?

### DISCUSSION THREAD

**Discussion by dougli1sqrd** [rustbox/discussion#16] [Ideas]
# Name thoughts?

So I think the name should be a made up compound work (like the way we write Rustbox as one word), and NOT camel cased like we're writing a java class and we're a tech company from 2009.

If the name should have more than one actual word, a space is sufficient. Bring back spaces in tech names!

# Brainstorming some names!

* Rustbox
* (The) Forest and (The) Tree(s)
* [something relating to gear? (cuz the rust gear logo is cool and bikes are cool?)]
* [something related to roses, to evoke worker solidarity/class consciousness? ]
* [something related to "open" (as in libre)?]
* Open Gear
* Electric Gear
* Electric Loom [loom to evoke a sense of historical literacy and connection to all the different crafts that brought us modern computing?]
* [Maybe on the historical trend, maybe there's a machine/industry that was critical in building the labor movement that we could riff on?]
* [This last point is reminding me of luddites in a good way, maybe a name could evoke big criticism of tech in the way the luddites did?]

Maybe we can kinda make a "name generator" by doing something like randomly picking two words together that we like and picking ones that sound nice?

**Comment by sethp** [comment]
I love these ideas! The forest/tree vibe feels especially like my jam, though I do also appreciate the "O[S]H" (Open [Source] Hardware) riffs too.

A few more loose words I have floating around in this space:

* silicon (si, 14th element)
    * also copper (cu, 29th element), tin (sn, 50th element), quartz (SiO4)   
* spicy / poisoned sand
    * also dopants, dopes,  maybe there's something there? two dopes in a glass house?
* magic smoke
* angry electrons
* spicy wires
* hot (metal) glue
* trees; collections thereof: stand, copse, thicket, grove; slightly more archaic would be weald or coppice 
   * other fun place names: glade, glen, dale, gorge, vale
* forestry / ecology
   * I like the care-taking and place-making angle this one offers, but I don't know a whole lot about either :grimacing: 
   * Silicon-Curious Arboreal Urbanists ? It's at least descriptive :upside_down_face: 

> NOT camel cased like we're writing a java class and we're a tech company from 2009.

Well if you're just gonna come for me like that, damn, ok. 

> Maybe we can kinda make a "name generator" by doing something like randomly picking two words together that we like and picking ones that sound nice?

This feels like it's 20° off-axis for an LLM, to me. We almost want a thing that's going to put together "nice-sounding, semi-coherent, but not entirely expected" combinations for us to look through, which is nearly what an LLM does ("nice-sounding, semi-coherent, entirely expected"). Maybe we could get close with a plain old markov chain ("usually incoherent, often surprising") especially if we sampled it in a, like, most-least-most likely sort of way? Or we could pull down one of the open LLM things, and see if we could finesse the sampler to be closer to what we want and less........ intentionally banal.

Plus, if we do it right, we could then pivot our name generator into a full-blown multi-billion-dollar business because we used an LLM, right?

**Comment by dougli1sqrd** [comment]
A name should:
* Evoke friendliness
* Openness
* Evoke curiosity
* Be unlike other tech org name
* Evoke community, togetherness

For example, Electric Embers is an interesting name. It's sort of long, invites curiosity to the people who would name themselves that. It's gentle and inviting.

Something we landed on at the end of our recent conversation:

**(The) Rose & Gear**, sort of trying to evoke old Inn/Tavern names "X & Y"

**Comment by dougli1sqrd** [comment]
* Terra Verde Cooperative
* Ember & Oak (Cooperative)
* TerraNova Cooperative
* Catalyst Collective
* Lichen Lab
* Root & Branch
* Moss Network
* Forge Collective
* Workshop Cooperative
* Commons Foundry
* Climate Craft Cooperative
* Open Source Ecology Coop
* Tech Commons Coop
* Green Byte Coop
* Community Forge

**Comment by dougli1sqrd** [comment]
Okay my final favorites of these:
* Terra Verde (Coop | Collective | <something else>)
* Ember & Oak (Coop | Collective | Lab | <something else>)
* Lichen (Lab | Workshop | <something else?>)
* Root & Branch <Noun>
* Ember & Rose (Coop | Collective | Lab | <something else>)

**Reply by dougli1sqrd** [reply]
Chewing on this some more, I think I have some favorite of favorites:
* Ember & (Rose | Oak) Coop
* Lichen Lab (Coop | Collective)?

**Comment by dougli1sqrd** [comment]
Adding a new entry lol
* Copper Rose
